OWML.ModHelper.Input/ModInputHandler.cs

```diff
@@ -285,14 +279,30 @@ private List<string> GetCollisions(ReadOnlyCollection<long> hashes)
                 {
                     combos.Add(_comboRegistry[hash].FullName);
                 }
-                if (hash < MaxUsefulKey && _gameBindingCounter[hash] > 0)
+                if (hash < ModInputLibrary.MaxUsefulKey && _gameBindingCounter[hash] > 0)
                 {
                     combos.Add("Outer Wilds." + Enum.GetName(typeof(KeyCode), (KeyCode)hash));
                 }
             }
             return combos;
         }
 
+        public List<string> GetCollisions(string combinations)
+        {
+            var hashes = new List<long>();
+            foreach (var combo in combinations.Split('/'))
+            {
+                var hash = ModInputLibrary.StringToHash(combo);
+                if (hash <= 0)
+                {
+                    var toReturn = new List<string>();
+                    toReturn.Add(((RegistrationCode)(-hash)).ToString());
```

Comment:
not used

Reply:
lol, I forgot to write 'return toReturn;'

Reply:
figured might as well turn it into one-liner